tools/src/main/java/org/apache/rocketmq/tools/command/metadata/RocksDBConfigToJsonCommand.java

```diff
@@ -47,41 +54,116 @@ public String commandName() {
 
     @Override
     public String commandDesc() {
-        return "Convert RocksDB kv config (topics/subscriptionGroups/consumerOffsets) to json";
+        return "Convert RocksDB kv config (topics/subscriptionGroups/consumerOffsets) to json. " +
+            "Use [-n, -c, -t] to send Request to broker ( version >= 5.3.2 ) or use [-p, -t, -e (not require)] load RocksDB. " +
+            "If -e is provided, tools will export json file instead of std print";
     }
 
     @Override
     public Options buildCommandlineOptions(Options options) {
         Option pathOption = new Option("p", "configPath", true,
                 "Absolute path to the metadata config directory");
-        pathOption.setRequired(true);
         options.addOption(pathOption);
 
         Option configTypeOption = new Option("t", "configType", true, "Name of kv config, e.g. " +
                 "topics/subscriptionGroups/consumerOffsets");
-        configTypeOption.setRequired(true);
         options.addOption(configTypeOption);
 
+        Option exportPathOption = new Option("e", "exportFile", true,
+            "Absolute file path for exporting, auto backup existing file, not directory!");
+        options.addOption(exportPathOption);
+
+        Option nameserverOption = new Option("n", "nameserverAddr", true,
+            "nameserverAddr. If nameserverAddr and clusterName are provided, will ignore [-p, -t, -e] args");
+        options.addOption(nameserverOption);
+
+        Option clusterOption = new Option("c", "cluster", true,
+            "Cluster name. If nameserverAddr and clusterName are provided, will ignore [-p, -t, -e] args");
+        options.addOption(clusterOption);
+
```

Comment:
Are there situations where it is necessary to use the -b parameter to specify an individual broker?